examples/tabular/20_basics/example_tabular_classification.py

```diff
@@ -22,32 +22,10 @@
 import sklearn.model_selection
 
 from autoPyTorch.api.tabular_classification import TabularClassificationTask
-from autoPyTorch.utils.hyperparameter_search_space_update import HyperparameterSearchSpaceUpdates
-
-
-def get_search_space_updates():
-    """
-    Search space updates to the task can be added using HyperparameterSearchSpaceUpdates
-    Returns:
-        HyperparameterSearchSpaceUpdates
-    """
-    updates = HyperparameterSearchSpaceUpdates()
-    updates.append(node_name="data_loader",
-                   hyperparameter="batch_size",
-                   value_range=[16, 512],
-                   default_value=32)
-    updates.append(node_name="lr_scheduler",
-                   hyperparameter="CosineAnnealingLR:T_max",
-                   value_range=[50, 60],
-                   default_value=55)
-    updates.append(node_name='network_backbone',
-                   hyperparameter='ResNetBackbone:dropout',
-                   value_range=[0, 0.5],
-                   default_value=0.2)
-    return updates
```

Comment:
Let's separate this PR from next time as well:)

Reply:
what do you mean?

Reply:
It is already a large PR, so it will take longer. If you just changed the doc-string, I think we could already merge it to the `refactor_development`.

Reply:
this PR is for changing the examples according to #111. We wanted to have a vanilla example for tabular classification, therefore I moved the search space updates to a different example file.